.travis.yml

```diff
@@ -10,14 +10,11 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
-before_install:
-  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
```

Comment:
ICU should still be required. Do builds on OSx succeed after this removal? Either way, we've been consistent in listing all requirements whether or not the current (Beta) Travis system includes it by default. See above `addons` list.